packages/core/app/src/initialize.ts

```diff
@@ -1,73 +1,105 @@
-import type {
-    Wallet,
-    Wallets,
-    WalletsCallback,
-    WalletsEventNames,
-    WalletsEvents,
-    WalletsWindow,
-} from '@wallet-standard/standard';
+import type { Wallet, WalletsCallback, WalletsWindow, Wallets } from '@wallet-standard/standard';
 
 declare const window: WalletsWindow;
 
+let initialized: InitializedWallets | undefined = undefined;
```

Comment:
I think this would be less confusing (eg. `initialized` here is not a boolean, despite being named bool-lean-ly) if this was called `api` or `cachedApi` and `WalletsAPI`.

Reply:
Yeah that's reasonable. Especially because the Wallets interface now has absolutely nothing to do with InitializedWallets.